.circleci/config.yml.in

```diff
@@ -263,14 +263,24 @@ jobs:
   prototype_test:
     docker:
       - image: circleci/python:3.7
+    resource_class: xlarge
     steps:
       - run:
           name: Install torch
-          command: pip install --user --progress-bar=off --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+          command: |
+            pip install --user --progress-bar=off numpy
+            pip install --user --progress-bar=off --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
       - run:
           name: Install prototype dependencies
           command: pip install --user --progress-bar=off git+https://github.com/pytorch/data.git
       - checkout
+      - run:
+          name: Download model weights
+          background: true
```

Comment:
This will spawn the step in the background. That means the job will not turn up in the CircleCI UI until it exits. Since downloading is a I/O bound operation, we can in install `torchvision` in the foreground, since that is mostly CPU bound.

Reply:
What would be the outcome of starting the tests prior restoring the weights? My understanding is that we will just redownload the weights one by one. Right? If that's the case, then we can leave as is.

If that's not the case then we should ensure this finishes prior running the tests. If the CircleCI doesn't allow us to do this via their API, we could use `bg`/`fg` to send things on background and then back on foreground. Unfortunately this solution is not going to be cross-platform and eventually we will need to run these tests on Windows as well.

Reply:
> My understanding is that we will just redownload the weights one by one. Right? If that's the case, then we can leave as is.

Correct. Although with the current timings we have very little risk to hit that case. In the [most recent CI run](https://app.circleci.com/pipelines/github/pytorch/vision/12908/workflows/691cdabd-3807-4ab7-986c-3c081d12f80c/jobs/1026896/steps), the download took 13 seconds whereas the installation took over 5 minutes.